bgpd/bgp_evpn.c

```diff
@@ -626,7 +626,7 @@ static int bgp_zebra_send_remote_macip(struct bgp *bgp, struct bgpevpn *vpn,
 	else {
 		ipa_len = is_evpn_prefix_ipaddr_v4(p) ? IPV4_MAX_BYTELEN
 						      : IPV6_MAX_BYTELEN;
-		stream_putl(s, ipa_len);
+		stream_putw(s, ipa_len);
```

Comment:
I guess you need to update the other case above to use `stream_putw(s, 0)` instead of `stream_putl(s, 0)` in order to fix the CI failure.